### ISSUE TITLE
Suggestions for the form

### ISSUE THREAD

**Issue by afeld** [18F/joining-18f#170]
Just read through the site thoroughly for the first time since the new application process was implemented – very glad that we're publishing so much information there! Jotted down a few things that I think could make the form better:
- [ ] Change the headings to be sentence case (per the [content guide](https://pages.18f.gov/content-guide/capitalization/)), and remove the colons.
- [ ] Make the "verify X" checkboxes required.
- [ ] Give some guidance about how much information we're looking for in the free-form questions.
- [ ] Add a note about the "government-style resume", and maybe link back to the [how to apply](https://pages.18f.gov/joining-18f/how-to-apply/) page.

Thanks!

**Comment by fureigh** [comment]
Thanks for these, @afeld! I was just about to open an issue about the "verify X" checkboxes.

@bill or @jamiealbrecht, what would you suggest for the third point ("guidance about how much information we're looking for in the free-form questions")?

**Comment by pixelwake** [comment]
@afeld @fureigh  We can add some guidance to those fields if we think it's necessary. What's the best way to draft up the content? Since it's the application, I'm not sure where to create the PR.

**Comment by afeld** [comment]
Yeah, it wouldn't be a pull request since it's in Google Docs...feel free to just check the simple things above, or propose changes here.

For the length guidance, I think something as simple as "a paragraph or two is fine" would work.

**Comment by fureigh** [comment]
@rooneywp The form is processed in Google Docs, but its markup is in [this file](https://github.com/18F/joining-18f/blob/18f-pages/pages/apply.html), so you can file a PR with text changes there if you'd like.

**Comment by fureigh** [comment]
Connecting #159 for resume clarification.

**Comment by fureigh** [comment]
@afeld Thanks again for these. With the switch to Lever in #189, the following are the case:
- Each position has its own application. (Here's a current [example](https://jobs.lever.co/18f/73bfd32b-ee89-4597-848f-745e58d11efd/apply).)
- Headings are all now in either all-caps or sentence case.
- The "verify X" checkboxes are now required.

These suggestions are still relevant:
- Give some guidance about how much information we're looking for in the free-form questions.
- Add a note about the "government-style resume", and maybe link back to the [how to apply](https://pages.18f.gov/joining-18f/how-to-apply/) page.

@rooneywp, do you have time to draft proposed text or ask the good folks of the Writing Lab to do so?

**Comment by jamiealbrecht** [comment]
Wanted to follow up on this issue because I am still not sure that folks are understanding "how to write a federal resume". I wonder if we might be able to write up some tips and tricks coming from 18F on how to do this?

**Comment by fureigh** [comment]
@jamiealbrecht Thanks for following up on this! Tips and tricks, and/or examples, would be great. Are you game to draft something? The good folks of the Writing Lab can likely help, but I think you're particularly well-suited, as someone who's worked in government before and sees a lot of resumes.
